0.25.0: Five failing tests

### Issue description

Hi :wave: 

I'm currently upgrading the package on Arch Linux to 0.25.0 and ran into failing tests:

```
=========================== short test summary info ============================
FAILED test/backend/x11/test_window.py::test_urgent_hook_fire[wayland-2] - Ke...
FAILED test/widgets/test_clock.py::test_clock_datetime_timezone - AssertionEr...
FAILED test/widgets/test_clock.py::test_clock_pytz_timezone - AssertionError:...
FAILED test/widgets/test_clock.py::test_clock_dateutil_timezone - AssertionEr...
FAILED test/widgets/test_clock.py::test_clock_change_timezones - AssertionErr...
= 5 failed, 1726 passed, 71 skipped, 4 xpassed, 2 warnings in 1017.94s (0:16:57) =
```

[qtile-0.25.0-1-x86_64-check.log](https://github.com/qtile/qtile/files/14965804/qtile-0.25.0-1-x86_64-check.log)
[qtile-0.25.0-1-x86_64-build.log](https://github.com/qtile/qtile/files/14965805/qtile-0.25.0-1-x86_64-build.log)


In regards to how tests are run, nothing has been changed since last upgrade:
https://gitlab.archlinux.org/archlinux/packaging/packages/qtile/-/blob/c9582a5362b017c90a6f3c06b57003dd1326901b/PKGBUILD

### Version

0.25.0

### Backend

X11 (default), Wayland (experimental)

### Config

_No response_

### Logs

_No response_

### Required

- [X] I have searched past issues to see if this bug has already been reported, and it hasn't been.
- [X] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

--- a/test/widgets/test_clock.py
+++ b/test/widgets/test_clock.py
@@ -43,6 +43,11 @@ class MockDatetime(datetime.datetime):
     def now(cls, *args, **kwargs):
         return cls(2021, 1, 1, 10, 20, 30)
 
+    def astimezone(self, tzone=None):
+        if tzone is None:
+            return self
+        return self + tzone.utcoffset(None)
+
 
 @pytest.fixture
 def patched_clock(monkeypatch):

